spark_log_parser/parsing_models/application_model_v2.py

```diff
@@ -27,22 +27,22 @@
 
 
 class SparkApplication:
-    # TODO - are these booleans actually necessary? Would `spark_app.sqlData is not None` suffice?
-    existsSQL: bool = False
-    sqlData: pd.DataFrame = None
 
-    existsExecutors: bool = False
-    executorData: pd.DataFrame = None
+    def __init__(self):
+        # TODO - are these booleans actually necessary? Would `spark_app.sqlData is not None` suffice?
+        self.existsSQL: bool = False
+        self.sqlData: pd.DataFrame = None
 
-    metadata: dict = {}
```

Comment:
I was a little worried about this one, though I think because we re-set `self.metadata`  instead of writing into the default value it ends up working fine... but still

Reply:
I wonder if static analysis tools like MyPy just care about the type that you declare in `__init__`